crates/polars-ops/src/series/ops/is_in.rs

```diff
@@ -373,17 +373,35 @@ pub fn is_in(s: &Series, other: &Series) -> PolarsResult<BooleanChunked> {
             is_in_boolean(ca, other)
         },
         DataType::Float32 => {
-            let other = other.cast(&DataType::Float32)?;
-            let other = other.f32().unwrap();
-            let other = other.reinterpret_unsigned();
+            let other = match other.dtype() {
+                DataType::List(_) => {
+                    let other = other.cast(&DataType::List(Box::new(DataType::Float32)))?;
+                    let other = other.list().unwrap();
+                    other.reinterpret_unsigned()
```

Comment:
TBH, I'm not sure if this(`reinterpret list`) is on the right way. And I have a question: will it produce unexpected results if we directly cast to `List<UInt32>`?

Reply:
If we cast they will round up. Floating point values in a hash operations are iffy at best. I decided to only support them by interpreting their bytes as is. We also do this in group_by with floating values.

Reply:
I see, make sense.